Update DEVELOPER.md (#5183)
Kong/kong: DEVELOPER.md

> These are the steps we follow in Kong to set up a development environment.

Reviewer: incorrect change

        --pcre 8.43
```

> After this task, we'd like to have the next steps to use the built packages and for LuaRocks to install new packages inside this `build` directory.  For that, it's important to set the `$PATH` variable accordingly:

Reviewer: ```suggestion
After this task, the following steps will use these built packages and LuaRocks will install new packages inside this `build` directory.  It is important to set the `$PATH` variable accordingly:
```

Author: done
